packages/dev/src/quickstarts-data/yaml/template.yaml

```diff
@@ -35,11 +35,11 @@ spec:
 
         1. The main body of the task. You can use markdown syntax here to create list items and more.
 
-          This is a paragraph.  
```

Comment:
funny enough the 2 spaces at the end of the line are intentional to force a line break :D alternatively you could insert an empty line between